spec/support/ruby_version_helpers.rb

```diff
@@ -0,0 +1,23 @@
+module RubyVersionHelpers
+  RUBY_2_4 = Gem::Version.new('2.4.0')
+  CURRENT_RUBY = Gem::Version.new(RUBY_VERSION)
+
+  def numeric_type
+    if CURRENT_RUBY < RUBY_2_4
+      Fixnum
+    else
+      Integer
+    end
+  end
+  module_function :numeric_type
+
+  def a_numeric
+    type = numeric_type
+    if type == Integer
```

Comment:
I'm not thrilled with this method. I would have liked to reuse the logic already in code to infer the appropriate article word. It wasn't immediately obvious how that might work, so I opted for this pragmatic solution. Happy to update if wanted!